src/grt/src/GlobalRouter.tcl

```diff
@@ -132,6 +132,18 @@ proc set_routing_layers { args } {
   }
 }
 
+sta::define_cmd_args "set_critical_nets_percentage" { percentage }
```

Comment:
Why is this a separate command rather than an argument to global_route?
I think the only reason to use separate commands is when the value is shared between multiple commands.
You need to update to README.md to doc this or another arg too.